src/components/Card.js

```diff
@@ -93,4 +108,16 @@ export class Card extends Component {
   }
 }
 
-export default Card;
+const mapStateToProps = state => {
+  const remainingActions = playerActions.getPlayerActions(state);
+  return { remainingActions };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    ...bindActionCreators({ ...playerActions }, dispatch),
+    dispatch
```

Comment:
Since we're mapping the actions to props we must also provide the `dispatch` action here otherwise we can't use it, and we'll need to use it to dispatch the actions that we receive in the actions prop.